src/Core/Framework/Adapter/Twig/SwTwigFunction.php

```diff
@@ -107,9 +126,12 @@ public static function escapeFilter(Environment $env, mixed $string, string $str
      * @param array<array-key, mixed> $context
      *
      * @return mixed
+     *
+     * @deprecated tag:v6.7.0 - Will be removed
```

Comment:
uff, why are the classes in the adapter namespace not all marked as "internal"? :scream: